modules/claims_api/app/models/claims_api/auto_established_claim.rb

```diff
@@ -53,6 +53,7 @@ class AutoEstablishedClaim < ApplicationRecord
     alias token id
 
     def to_internal
+      form_data['applicationExpirationDate'] ||= build_application_expiration
```

Comment:
should we be throwing away what the consumer submitted? (if they submitted of course)
im torn because it sounds like it doesnt really matter but also it seems wrong to toss aside something the consumer submitted to us 🤷🏻